pages/common/magick.md

```diff
@@ -10,7 +10,7 @@
 
 - Resize an image, making a new copy:
 
-`magick convert -resize {{100x100}} {{image.jpg}} {{image.jpg}}`
+`magick {{image1.jpg}} -resize {{100x100}} {{image2.jpg}}`
```

Comment:
```suggestion
`magick {{input_image.jpg}} -resize {{100x100}} {{output_image.jpg}}`
```

Reply:
In addition to this, I think we can use the path placeholder across the page for image files.